src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -2171,7 +2171,7 @@ impl Coordinator {
     ) {
         // Process the current stage and allow for processing the next.
         loop {
-            event!(Level::TRACE, stage = format!("{:?}", stage));
+            // event!(Level::TRACE, stage = format!("{:?}", stage));
```

Comment:
I'll have to figure out why `Level::TRACE` is logged in CI and revert this commit before we merge this.